go.mod

```diff
@@ -8,17 +8,17 @@ require (
 	github.com/otiai10/copy v1.0.2
 	github.com/pkg/errors v0.9.1
 	github.com/snikch/goodman v0.0.0-20171125024755-10e37e294daa
-	github.com/spf13/cobra v0.0.6
-	github.com/spf13/viper v1.6.2
+	github.com/spf13/cobra v1.0.0
```

Comment:
This seems a big jump has this come in via the consensus?

Reply:
yep: https://github.com/fetchai/cosmos-consensus/pull/248/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R25